### PR TITLE
Ensure we can get latest timestamp

### PR DESCRIPTION
**Pull request by lassemand**
## Purpose

I want to be able to trace whether the notification service is still successfully processing blocks

## Checklist

- [x] My code follows the style of this project.
- [x] The code compiles without warnings.
- [x] I have performed a self-review of the changes.
- [x] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [x] (If necessary) I have updated the CHANGELOG.

### CODE REVIEW

File: notification-server/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
+## 0.3.3
+- Add latest successful processed block
```
Comment (soerenbf) [inline notification-server/CHANGELOG.md:2]: This is not really what happens though? 🤷 You trace when you last processed a block successfully. And I don't think this is the best solution for the purpose described in the PR description

File: notification-server/src/bin/service.rs [hunk 2]
```diff
@@ -329,6 +329,10 @@ async fn traverse_chain(
                 processed_height = block_height;
                 let delta = start.elapsed();
                 histogram!("block.process_successful_duration").record(delta);
+                if let Ok(duration_since_epoch) = SystemTime::now().duration_since(UNIX_EPOCH) {
+                    let current_timestamp = duration_since_epoch.as_secs() as f64;
+                    gauge!("block.process_successful_last_unix").set(current_timestamp);
+                }
```
Comment (soerenbf) [inline notification-server/src/bin/service.rs:335]: If your goal is to trace when the service is down, you could do it in a more precise and efficient (+ configurable) way I'd say. Something like

```rust
struct Args {
  ...
  alive_timeout_seconds: u8,
}

async fn main() {
  tokio::select! {
    _ = liveness_process(args.alive_timeout_seconds, ...) => {}
    _ = chain_traversal(...) => {}
  }
}
```

Comment (lassemand) [reply notification-server/src/bin/service.rs:335]: I did write an invalid title. The purpose is not to determine whether the service is up or down, but instead to trace whether it is successfully processing blocks. If no blocks has been successfully processed within the last say 5m then I will emit an alert. Much like this: 

https://metrics.internal.testnet.concordium.com/d/JWAZ2MiSk/alerts-non-critical?orgId=1&viewPanel=59

File: notification-server/CHANGELOG.md (hunk 1 above)
Comment (soerenbf) [inline notification-server/CHANGELOG.md:2]:
```suggestion
- Add timestamp for when a block was last processed to tracked metrics
```

**Review by soerenbf — APPROVED** [review]

**Pull request merged**